plugins/catalog-backend-module-msgraph/README.md

```diff
@@ -56,6 +56,9 @@ catalog:
           # See https://docs.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-1.0#properties
           # and for the syntax https://docs.microsoft.com/en-us/graph/query-parameters#filter-parameter
           userFilter: accountEnabled eq true and userType eq 'member'
```

Comment:
Maybe this is a silly question but is there no other filter you could apply to users to at least narrow down the search set to something more reasonable? For example checking the `department`?

Reply:
Not really, unfortunately the AD hasn't been setup in a way where I could use something like that, unless I let the user filter grow to many lines. Also, `department` is something I tried using but required users are in different departments and department names have changed multiple times recently because of org restructure. There isn't much else available to use. MSGraph API doesn't help much because of it's filter limitations and reworking AD just so that we can load users to backstage isn't a really strong case.